ts/src/okx.ts

```diff
@@ -759,7 +760,7 @@ export default class okx extends Exchange {
                 'fetchPositions': {
                     'method': 'privateGetAccountPositions', // privateGetAccountPositions or privateGetAccountPositionsHistory
                 },
-                'createOrder': 'privatePostTradeBatchOrders', // or 'privatePostTradeOrder' or 'privatePostTradeOrderAlgo'
+                'createOrder': 'privatePostTradeBatchOrders', // privateGetAccountBills or 'privatePostTradeOrder' or 'privatePostTradeOrderAlgo'
```

Comment:
@ttodua how is this method related to `createOrder`?

Reply:
uhoh, i misplaced that comment when pasting. fixed now